x-pack/plugins/lens/public/xy_visualization/xy_expression.tsx

```diff
@@ -389,18 +434,11 @@ export function XYChart({
           id={axis.groupId}
           groupId={axis.groupId}
           position={axis.position}
-          title={
-            axis.series
-              .map(
-                (series) =>
-                  data.tables[series.layer].columns.find((column) => column.id === series.accessor)
-                    ?.name
-              )
-              .filter((name) => Boolean(name))[0] || args.yTitle
-          }
-          showGridLines={false}
+          title={showYAxisTitle ? getYAxesTitles(axis.series, index) : undefined}
+          showGridLines={gridlinesVisibilitySettings?.y}
           hide={filteredLayers[0].hide}
-          tickFormat={(d) => axis.formatter.convert(d)}
+          // @ts-ignore, temporary solution for not displaying the ticks
+          tickFormat={tickLabelsVisibilitySettings?.y ? (d) => axis.formatter.convert(d) : () => {}}
```

Comment:
Can we replace this by ` () => ''` - this should satisfy the type (so we don't need to ts-ignore), but still render no ticks, right?